src/core/database/transaction.ts

```diff
@@ -136,8 +136,9 @@ export class Transaction implements QueryConnection {
    * Streaming is not supported on transactions
    * This observable will immediately emit an error.
    */
+  // @ts-ignore FIXME: unclear how to get around the TS error here
   stream<R = any>(query: Query): Observable<Dictionary<R>> {
```

Comment:
Is this the `any-observable` problem?

Reply:
Ok yeah it is. Let's just replace the any-observable import with rxjs.
```diff
-import Observable from 'any-observable';
+import { Observable } from 'rxjs';
```
That should fix the need to explicitly define the `subscriber` parameter too.
And we won't need the `declare module` stub.

Reply:
Yeah, `any-observable` gave me some grief here. I stubbed it in `typings.d.ts`, even exported it as type `any` at one point. We may need to define it further to eliminate the error being thrown here. It's complaining that `Observable` cannot be used as a type here.

Reply:
Did you see my second comment?

Reply:
Sorry about that, GH page was cached. Switching to rxjs worked!

Reply:
No worries. Happens to me all the time. GitHub updates some messages but misses others. I refresh all the time out of paranoia lol